playbooks/adhoc/upgrades/upgrade.yml

```diff
@@ -52,6 +90,14 @@
     fail: This playbook requires Origin 1.0.6 or Atomic OpenShift 3.0.2 or later
     when: _new_version.stdout | version_compare('1.0.6','<') or ( _new_version.stdout | version_compare('3.0','>=' and _new_version.stdout | version_compare('3.0.2','<') )
 
+- name: Verify upgrade can proceed
```

Comment:
I think we can move this, "Ensure AOS 3.0.2 or Origin 1.0.6", and "Determine new master version" to right after the base package upgrade and then use the version of the base package for comparisons.